near/test/sdk.ts

```diff
@@ -0,0 +1,874 @@
+  SuggestedParams,
+  bigIntToBytes,
+  decodeAddress,
+  getApplicationAddress,
+  makeApplicationCallTxnFromObject,
+  makePaymentTxnWithSuggestedParamsFromObject,
+  waitForConfirmation,
+} from "@certusone/wormhole-sdk/node_modules/algosdk";
+
+import {
+  createAsset,
+  getAlgoClient,
+  getBalance,
+  getBalances,
+  getForeignAssetFromVaaAlgorand,
+  getTempAccounts,
+  signSendAndConfirmAlgorand,
+} from "./algoHelpers";
+
+import {
+  CHAIN_ID_ALGORAND,
+  CHAIN_ID_NEAR,
+  ChainId,
+  ChainName,
+  textToHexString,
+  textToUint8Array,
+} from "@certusone/wormhole-sdk/lib/cjs/utils";
+
+import { safeBigIntToNumber } from "@certusone/wormhole-sdk/lib/cjs/utils/bigint";
+
+import {
+  CONTRACTS,
+  attestNearFromNear,
+  attestTokenFromNear,
+  attestFromAlgorand,
+  createWrappedOnAlgorand,
+  createWrappedOnNear,
+  getEmitterAddressAlgorand,
+  getForeignAssetAlgorand,
+  getForeignAssetNear,
+  getIsTransferCompletedNear,
+  getIsWrappedAssetNear,
+  getOriginalAssetNear,
+  getSignedVAAWithRetry,
+  redeemOnAlgorand,
+  redeemOnNear,
+  transferFromAlgorand,
+  transferNearFromNear,
+  transferTokenFromNear,
+} from "@certusone/wormhole-sdk";
+
+const wh = require("@certusone/wormhole-sdk");
+
+import { parseSequenceFromLogAlgorand } from "@certusone/wormhole-sdk/lib/cjs/bridge";
+
+import {
+  getMessageFee,
+  optin,
+  TransactionSignerPair,
+  _parseVAAAlgorand,
+} from "@certusone/wormhole-sdk/lib/cjs/algorand";
+
+export const uint8ArrayToHex = (a: Uint8Array): string =>
+  Buffer.from(a).toString("hex");
+
+export const hexToUint8Array = (h: string): Uint8Array =>
+  new Uint8Array(Buffer.from(h, "hex"));
+
+function getConfig(env: any) {
+  switch (env) {
+    case "sandbox":
+    case "local":
+      return {
+        networkId: "sandbox",
+        nodeUrl: "http://localhost:3030",
+        masterAccount: "test.near",
+        wormholeAccount: "wormhole.test.near",
+        tokenAccount: "token.test.near",
+        userAccount:
+          Math.floor(Math.random() * 10000).toString() + "user.test.near",
+        user2Account:
+          Math.floor(Math.random() * 10000).toString() + "user.test.near",
+      };
+  }
+  return {};
+}
+
+export function logNearGas(result: any, comment: string) {
+  const { totalGasBurned, totalTokensBurned } = result.receipts_outcome.reduce(
+    (acc: any, receipt: any) => {
+      acc.totalGasBurned += receipt.outcome.gas_burnt;
+      acc.totalTokensBurned += nearUtils.format.formatNearAmount(
+        receipt.outcome.tokens_burnt
+      );
+      return acc;
+    },
+    {
+      totalGasBurned: result.transaction_outcome.outcome.gas_burnt,
+      totalTokensBurned: nearUtils.format.formatNearAmount(
+        result.transaction_outcome.outcome.tokens_burnt
+      ),
+    }
+  );
+  console.log(
+    comment,
+    "totalGasBurned",
+    totalGasBurned,
+    "totalTokensBurned",
+    totalTokensBurned
+  );
+}
+
+export function parseSequenceFromLogNear(result: any): [number, string] {
```

Comment:
Didn't you already write this in the sdk?

Reply:
yeah,  I probably wrote it here and then moved it over to the sdk and could delete it from here